### ISSUE TITLE
[BUG] PlaceholderAPI's expansion ServerExpansion do not loads.

### ISSUE THREAD

**Issue by Jagerente** [magmafoundation/Magma#456]
PlaceholderAPI's expansion - ServerExpansion doesn't loads.

Magma-99be8c5-STABLE-server
PlaceholderAPI-2.10.10
`/papi ecloud download Server`

![image](https://user-images.githubusercontent.com/47350537/127241895-5e2fa919-fd50-4715-b048-664afe0a7fa4.png)

**Comment by DrakstaarMSD** [comment]
I also tried using PlacehoderAPI and got it to work as long as I didn't add any expansions.
When I added expansions and restarted the server the server would get hung up at registering the expansion.

**Comment by Jagerente** [comment]
> 
> 
> I also tried using PlacehoderAPI and got it to work as long as I didn't add any expansions.
> When I added expansions and restarted the server the server would get hung up at registering the expansion.
> The server wouldn't crash it just wouldn't continue.

There are actually some expansions that work fine with magma `playerlist` for example.

**Comment by 27638166** [comment]
vault expansion same here

**Comment by DrakstaarMSD** [comment]
> > I also tried using PlacehoderAPI and got it to work as long as I didn't add any expansions.
> > When I added expansions and restarted the server the server would get hung up at registering the expansion.
> > The server wouldn't crash it just wouldn't continue.
> 
> There are actually some expansions that works fine with magma `playerlist` for example.

Going to test some today  that I use see witch ones don't work.

**Comment by Flarkinater** [comment]
I did some testing and found that the bug appears only after update a2b8e25. That was a big update: https://github.com/magmafoundation/Magma/compare/v48ec0a1...va2b8e25-CUSTOM

**Comment by Jagerente** [comment]
Any fixes coming?

**Comment by Flarkinater** [comment]
@Jagerente I know this might seem unrelated, but do you use Essentials or CMI?

**Comment by Flarkinater** [comment]
After quite a bit of testing there are two different bugs related to PAPI:
* The originally reported one: Several different PAPI expansions fail to load, due to the error from the screenshot above. This happens _only_ on the latest PAPI version 2.10.10, regardless of what Magma version you use. So this might not be a Magma bug!
* The one mentioned by @DrakstaarMSD. I made a new ticket for that here: https://github.com/magmafoundation/Magma/issues/456
